s3_website.yml

```diff
@@ -1,6 +1,6 @@
 s3_id: <%= ENV['AWS_ACCESS_KEY_ID'] %>
 s3_secret: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
-s3_bucket: opengazettes.or.ke
+s3_bucket: nigeria.opengazettes.co
```

Comment:
@DavidLemayian Thanks. That's the config file I've been failing to change. I've moved the setting to use the environment variable set from  the `.env` file